ui/components/multichain/import-tokens-popover/import-tokens-popover.scss

```
@@ -0,0 +1,53 @@
+.import-tokens-popover {
+  $self: &;
+
+  .tabs {
+    ul {
+      border-bottom: 0;
+    }
+
+    li {
+      width: 50%;
+    }
+  }
+
+  &__search-list {
+    max-width: 260px;
```

static pixel values are a red flag for me because it prevents responsiveness in some cases, does this still work if it's 
```suggestion
    max-width: 100%;
```